src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -30,6 +30,7 @@ class Yaml
     const DUMP_MULTI_LINE_LITERAL_BLOCK = 128;
     const PARSE_CONSTANT = 256;
     const DUMP_EMPTY_ARRAY_AS_SEQUENCE = 1024;
+    const PARSE_KEYS_AS_STRING = 2048;
```

Comment:
But then the file is invalid, so shouldn't we consider it must be updated?
If someone has the case why not but it seems a bit too early imo to add this without being sure it will be used.